assessments/projects/cubefs/self-assessment.md

```diff
@@ -0,0 +1,257 @@
+# CubeFS Self-assessment
+This Self-assessment document has been created by students at NYU Tandon to help provide information regarding CubeFS, determining gaps in their security, and preparing any security
+documentation for their users. 
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+* [Action Items](#action-items)
+
+## Metadata
+
+|   |  |
+| -- | -- |
+| Software | https://github.com/cubefs/cubefs  |
+| Website | https://cubefs.io/ |
+| Security Provider | No  |
+| Languages | Golang |
+| SBOM | Currently, no SBOM is publicly available for CubeFS|

```

Comment:
Recommended minor change: highlight that target state is not yet achieved, while removing red-herring questions about a potential privately available SBOM.
 
```suggestion
| SBOM | CubeFS does not currently generate an SBOM on release |
```

Reply:
@eddie-knight, we have added the change you suggested. Please let us know if it's okay.